docs/API/helpers/quickstart.md

```diff
@@ -24,7 +24,10 @@ Fliplet.Helper({
       { name: 'title', type: 'text', label: 'Title' },
       { name: 'content', type: 'text', label: 'Content' }
     ]
-  }
+  },
+  // Enable support for reactive data when used
+  // in a dynamic container or list repeater component.
+  supportsDynamicContext: true
```

Comment:
Is there any downside or performance issues if all the helpers have this enabled, but don't actually use any dynamic context? If so, I suggest we use `false` in the example because most people copy and paste our provided examples and only change what they feel they need to.

```suggestion
  supportsDynamicContext: false
```